src/main/ngapp/accio/src/app/editor/magic-wand.service.ts

```diff
@@ -255,16 +165,16 @@ export class MagicWandService {
    */
   doFloodfill(imgData: ImageData, xCoord: number, yCoord: number,
       tolerance: number, scribbles: Set<number>): Set<number> {
+    // Creates an array of color objects from the indices in 'scribbles'.
+    // Color object: {red: number, green: number, blue: number}
+    const colors: Array<Color> = this.scribblesToColors(scribbles, imgData);
```

Comment:
I think you should use a more descriptive variable name here.  Something like: scribblesColors or thresholdColors etc.